cetlvast/suites/unittest/test_any.cpp

```diff
@@ -355,7 +343,7 @@ TEST(test_any, any_cast_cppref_example)
     EXPECT_STREQ("hollo", any_cast<const std::string&>(a1).c_str());  //< const reference
 
     auto s1 = any_cast<std::string&&>(std::move(a1));  //< rvalue reference
-    // Note: “s1” is a move-constructed std::string, “a1” is empty
+    // Note: `s1` is a move-constructed std::string, `a1` is empty
```

Comment:
We agreed to add back ticks "`" as an exception (b/c strictly speaking they are not allowed as well).